src/low-level/imap/mailimap.h

```diff
@@ -803,6 +803,10 @@ LIBETPAN_EXPORT
 void mailimap_set_logger(mailimap * session, void (* logger)(mailimap * session, int log_type,
     const char * str, size_t size, void * context), void * logger_context);
 
+#ifndef ENABLE_163_WORKAROUND
```

Comment:
Could you rename it to LIBETPAN_HAS_MAILIMAP_163_WORKAROUND?
It would then be similar to LIBETPAN_MAILSTREAM_NETWORK_DELAY.